src/test/java/com/rocketden/main/api/GameTests.java

```diff
@@ -124,7 +124,7 @@ private ProblemDto createSingleProblemAndTestCases() throws Exception {
         createProblemRequest.setName(NAME);
         createProblemRequest.setDescription(DESCRIPTION);
         createProblemRequest.setDifficulty(ProblemDifficulty.EASY);
-
+        createProblemRequest.setApproval(true);
```

Comment:
nit: here and for the other 1-3 createSingleProblem methods, could you add at the bottom something like `assertTrue(response.getApproval)`? Just to verify that this was properly set in the database

Reply:
k